spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe 'user/show', type: :system do
+  before(:each) do
+    driven_by(:rack_test)
+    @user = User.first
+  end
+
+  it 'has username' do
+    visit user_path(@user)
+    expect(page).to have_content(@user.name)
+  end
+
+  it 'has profile picture' do
+    visit user_path(@user)
+    expect(page).to have_css('img')
+  end
+
+  it 'has number of posts' do
+    visit user_path(@user)
+    expect(page).to have_content("Number of Posts: #{@user.posts_count}")
+  end
+
+  it 'has bio' do
+    visit user_path(@user)
+    expect(page).to have_content("Bio: #{@user.bio}")
+  end
+
+  it 'button to show more' do
+    visit user_path(@user)
+    expect(page).to have_content('Show more')
+  end
```

Comment:
Please add the following tests because you forgot to add them:
- [ ] a test that asserts you can see the user's first 3 posts as required [HERE](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=I%20can%20see%20the%20user%27s%20first%203%20posts.)
- [ ] a test that redirects to the post's show page when you click on a user's post as required [HERE](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=When%20I%20click%20a%20user%27s%20post%2C%20it%20redirects%20me%20to%20that%20post%27s%20show%20page.)
- [ ] a test that redirects to the user's post's index page when you click on a **see all posts** as required [HERE](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=When%20I%20click%20to%20see%20all%20posts%2C%20it%20redirects%20me%20to%20the%20user%27s%20post%27s%20index%20page.).